_posts/2021-04-28-#17331.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: pr
+date: 2021-04-28
+title: "Use effective values throughout coin selection"
+pr: 17331
+authors: [achow101]
+components: ["wallet"]
+host: Xekyo
+status: upcoming
+commit: 4ac1add
+---
+
+## Notes
+
+- Wallets maintain their balance as a pool of Unspent Transaction Outputs
+  (UTXOs).
+
+- _Coin selection_ refers to the process of picking UTXOs from the wallet's UTXO
+  pool to fund a transaction.
+
+- What are the goals of coin selection? Economics, privacy, reliability of
+  inputs/conflicts
```

Comment:
It's a bit confusing to form this as a question. Better would be a statement:

```suggestion
- The goals of coin selection are:
    - Minimizing fees
    - Maintaining privacy
    - Ensuring reliability of inputs/reducing the chance of transactions being conflicted.
```

Reply:
Filled out that section